content/docs/04-clusters/06-cluster-management.md

```diff
@@ -57,7 +57,11 @@ The following sections describe these capabilities in detail:
 * [Web based kubectl](/clusters/cluster-management/palette-webctl#overview) - Palette leverages Kubectl through an in-built command line interface for the users to communicate with their workload clusters.
 
 
-* [Lock Cluster](/clusters/cluster-management/palette-lock-cluster) - Palette supports the Cluster(s) Lock feature to exclude a cluster or a group of clusters from getting upgraded when Palette is upgraded. 
+* [Platform Management](/clusters/cluster-management/palette-lock-cluster) - Palette supports the Cluster(s) Lock feature to exclude a cluster or a group of clusters from getting upgraded when Palette is upgraded. 
```

Comment:
We should avoid word `Lock` rather should stick to Pause Updates of Platform components